.env

```diff
@@ -0,0 +1,11 @@
+DOCKER_INFLUXDB_INIT_MODE=setup
```

Comment:
Is there a reason why this has to be `.env`? I would prefer `test.env` or `demo.env`.